### PR TITLE
Dictionary icons not visible when window width not big enough (smartphone screens)

### PR DESCRIPTION
**Pull request by akanex115**
# Background
https://github.com/ajktown/wordnote/issues/166

## TODOs
- [x] Fix so that it shows every language dictionary icons

## Checklist (Developers)
- [x] `Draft` is set for this PR
- [x] `Title` is checked
- [x] `Background` is filled
- [x] `TODOs` are filled
- [x] `Assignee` is set
- [x] `Labels` are set
- [x] `Project` is created & linked, if multiple PRs are associated to this PR
- [x] `development` is linked if related issue exists
- [x] `yarn inspect` is run
- [x] `TODOs` are handled and checked
- [x] Final Operation Check is done
- [x] Make this PR as an open PR

## Checklist (Reviewers)
- [x] Check if there are any other missing TODOs that are not yet listed
- [x] Review Code
- [x] Every item on the checklist has been addressed accordingly
- [x] If `development` is associated to this PR, you must check if every TODOs are handled

### CODE REVIEW

**Review by mlajkim — COMMENTED** [review]
image

Hmm I dont think it works
Also the code could be refactored a bit more

**Comment by mlajkim** [comment]
@akanex115 
Reference: https://github.com/ajktown/wordnote/compare/main...using-stack

**Comment by akanex115** [comment]
Handled in [8703e54](https://github.com/ajktown/wordnote/pull/198/commits/8703e547cb5a10a1dceab7764d0444ec3a7c5ead)

File: src/components/molecule_word_card/index.tsx [hunk 1]
```diff
@@ -68,14 +76,27 @@ const WordCard: FC<Props> = ({ wordId, editingMode }) => {
...
         <CardActions>
-          <WordCardFavoriteIcon wordId={wordId} />
-          <WordCardDeleteButton wordId={wordId} />
-          {!word.isArchived && <WordCardArchiveButtonPart wordId={wordId} />}
-          {word.isArchived && <WordCardUnarchiveButtonPart wordId={wordId} />}
-          <WordCardSearchThisWordButtonPart wordId={wordId} />
-          <WordCardShareButtonPart wordId={wordId} />
-          <TagButtonChunk word={word} />
-          <DictLinkButtonChunk wordId={wordId} />
+          <Stack direction={width > 622 ? `row` : `column`} alignItems={`left`}>
```
Comment (mlajkim) [inline src/components/molecule_word_card/index.tsx:79]: I think we can use a bit simpler number like `630`?

File: src/components/molecule_word_card/index.tsx [hunk 2]
```diff
@@ -68,14 +76,27 @@ const WordCard: FC<Props> = ({ wordId, editingMode }) => {
...
+            <Stack
+              direction={width > 440 ? `row` : `column`}
+              alignItems={`left`}
+            >
+              <Stack direction={`row`} alignItems={`center`}>
```
Comment (mlajkim) [inline src/components/molecule_word_card/index.tsx:84]: Did you also check out how to refactor this cleaner with less indents?

**Review by mlajkim — COMMENTED** [review]
First review

**Review by mlajkim — APPROVED** [review]
OK

**Pull request merged**